lib/services/project-changes-service.ts

```diff
@@ -250,10 +252,17 @@ export class ProjectChangesService implements IProjectChangesService {
 					return true;
 				}
 			}
+
 		}
 		return false;
 	}
 
+	private isDirectoryModified(dirPath: string): boolean {
+		const dirPathStat = this.$fs.getFsStats(dirPath);
```

Comment:
The code is the same as the one used on line 235. Also the method is called `isDirectortyModified`, but the implementation does not have any code specific to directories only.
I suggest renaming the method and use it on line 235.